src/plugins/simulator/visualizations/qt-opengl/qtopengl_obj_model.cpp

```diff
@@ -315,6 +315,10 @@ namespace argos {
             }
             itMaterial = m_mapMaterials.emplace(strMaterialName.toStdString(), SMaterial()).first;
          }
+         else if(cLineBufferSplit[0] == "Ks") {
+	 itMaterial->second.Emission = {
+               0.0f, 0.0f, 0.0f, 1.0f};
+	 }
```

Comment:
As far as I can tell, the effect of this statement effectively just blocks Ks from being parsed below, leaving std::array<GLFloat, 4> Emission at its initialized state. If this is the intended effect, why not just remove both Ks statements?